Makefile

```diff
@@ -29,7 +29,7 @@ SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
 .PHONY: all
-all: tidy fmt vet lint schema-gen build build-cert build-cli test
```

Comment:
The issue is gone, but calling `go vet` is not necessary, because it is already integrated into `golangci-lint`. So we effectively called it twice every time.